api/src/test/scala/hmda/api/http/institutions/UploadPathsSpec.scala

```diff
@@ -38,24 +37,27 @@ class UploadPathsSpec extends InstitutionHttpApiAsyncSpec with SubmissionProtoco
   val repository = new LarRepository(config)
   val modifiedLarRepository = new ModifiedLarRepository(config)
 
+  private def dropAllObjects: Future[Int] = {
+    val db = repository.config.db
+    val dropAll = sqlu"""DROP ALL OBJECTS"""
+    db.run(dropAll)
+  }
+
   override def beforeAll(): Unit = {
     super.beforeAll()
-    dropAllObjects()
-    Await.result(repository.createSchema(), duration)
-    Await.result(modifiedLarRepository.createSchema(), duration)
+    val setup = for {
+      _ <- dropAllObjects
+      _ <- repository.createSchema()
+      c <- modifiedLarRepository.createSchema()
+    } yield c
+    Await.result(setup, duration)
   }
 
   override def afterAll(): Unit = {
     super.afterAll()
-    dropAllObjects()
-    repository.config.db.close()
```

Comment:
FYI, for those interested: This was the line that was causing the errors. This test would close the db, then any specs that ran after this one wouldn't be able to connect to it.